common/orchestra.py

```diff
@@ -103,24 +103,32 @@ def collapse(self, var, tabs=0):
             result += str(var.encode('utf-8'))
         return result
 
-    def export_info(self):
+    def export_info(self, f):
         # Date and Time for export File
         current_date = time.strftime("%m/%d/%Y").replace("/", "")
         current_time = time.strftime("%H:%M:%S").replace(":", "")
+
+        if f == "":
+            f = 'export_' + current_date + '_' + current_time + '.csv'      
+
         # True for printing the header on the first system
         # after that, only values
         add_header = True
 
         for path, ip_objd in self.system_objects.iteritems():
             attrs = vars(ip_objd[0])
-            with open('export_' + current_date + '_' + current_time + '.csv', 'a') as export_file:
-                csv_file = csv.DictWriter(export_file, attrs.keys())
-                if add_header:
-                    csv_file.writeheader()
-                    add_header = False
-                csv_file.writerow(attrs)
-
-        print helpers.color("\nExport file saved to disk at export_" + current_date + "_" + current_time + ".csv")
+            try:
+                with open(f, 'a') as export_file:
+                    csv_file = csv.DictWriter(export_file, attrs.keys())
+                    if add_header:
+                        csv_file.writeheader()
+                        add_header = False
+                    csv_file.writerow(attrs)
+                    print helpers.color("\nExport file saved to disk at " + f)
```

Comment:
Can you move this print statement out of the for loop?  Right now it prints to the console for each IP loaded into Just-Metadata.